### PR TITLE
`Development`: Add hand-in early with continue and reload page e2e exam tests

### PR DESCRIPTION
**Pull request by TheZoker**
#### General

- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).

### Description

With this PR we add new e2e exam participation tests, the test the following cases:

- Hand in early, but then decide to continue instead and change the submission text
- Reload the exam page in the middle of the exam and check if the content of the text exercise is still correct
- Reload the exam result page and check if all the content is still the same

#### Code Review
- [ ] Review 1
- [ ] Review 2

### CODE REVIEW

File: src/test/cypress/e2e/exam/ExamParticipation.cy.ts [hunk 1]
```diff
@@ -5,13 +5,14 @@ import submission from '../../fixtures/exercise/programming/all_successful/submi
 import { Course } from 'app/entities/course.model';
 import { generateUUID } from '../../support/utils';
 import { EXERCISE_TYPE } from '../../support/constants';
-import { courseManagementRequest, examExerciseGroupCreation, examNavigation, examParticipation, examStartEnd } from '../../support/artemis';
-import { Exercise } from 'src/test/cypress/support/pageobjects/exam/ExamParticipation';
+import { courseManagementRequest, examExerciseGroupCreation, examNavigation, examParticipation, examStartEnd, textExerciseEditor } from '../../support/artemis';
+import { AdditionalData, Exercise } from 'src/test/cypress/support/pageobjects/exam/ExamParticipation';
```
Comment (bassner) [inline src/test/cypress/e2e/exam/ExamParticipation.cy.ts:9]: Is this client style issue correct? Do we need this import?

**Review by bassner — CHANGES_REQUESTED** [review]

File: src/test/cypress/e2e/exam/ExamParticipation.cy.ts (hunk 1 above)
Comment (TheZoker) [reply src/test/cypress/e2e/exam/ExamParticipation.cy.ts]: Ah good point, I think that is a left over from the rebase. Thanks for the hint!

**Review by JohannesStoehr — APPROVED** [review]
Logic of new tests look good to me

**Review by Vinicius-Agreste — APPROVED** [review]
Tested on ts2. Working as expected.

**Pull request merged**